pyphare/pyphare/simulator/simulator.py

```
@@ -113,7 +113,7 @@ def setup(self):
                 self.cpp_hier,
             )
             return self
-        except:
+        except (Exception, ...):
```

Ruff

`except` handlers should only be exception classes or tuples of exception classes

(B030)